src/components/Table/GridTable.stories.tsx

```diff
@@ -2144,3 +2146,112 @@ export function MinColumnWidths() {
     </div>
   );
 }
+
+enum EditableRowStatus {
+  Active = "Active",
+  Inactive = "Inactive",
```

Comment:
Is this enum supposed to be actively changing the coloring/styling/behavior of the style, or is it just a dummy enum to show what a select field looks like in the table?

If it's just a dummy enum, can we make the enum values something like `Foo = "Foo"` and `Bar = "Bar`"` just to highlight their meaningless-ness?